commands/pipelines/setup.js

```diff
@@ -177,55 +187,61 @@ module.exports = {
     const github = new GitHubAPI(context.version, yield getGitHubToken(kolkrabbi))
 
     const organization = context.flags.organization || context.flags.team
-    const { name: pipelineName, repo: repoName } = yield getNameAndRepo(context.args)
-    const repo = yield getRepo(github, repoName)
-    const settings = yield getSettings(repo.default_branch)
-
-    let ciSettings
-    if (yield hasCIFlag(heroku)) {
-      ciSettings = yield getCISettings(organization)
+    const { name: pipelineName } = yield getPipelineName(context.args)
+    const stagingAppName = pipelineName + STAGING_APP_INDICATOR
+
+    if (stagingAppName.length > 30) {
+      cli.error(`${pipelineName} is too long.\nPlease, choose a pipeline name shorter than ${30 - STAGING_APP_INDICATOR.length} characters.`)
+    } else {
```

Comment:
Its fussy, but I'd prefer a `return` statement in the `if` branch so we don't have to wrap the rest of the command in an `else`